api-update/Jenkinsfile-test

```diff
@@ -0,0 +1,76 @@
+def versionTag = "v${env.BUILD_NUMBER}"
+
+pipeline {
+    agent {
+        label "master"
+    }
+    parameters {
+        choice(name: "STAGE", choices: ["intg", "staging", "prod"], description: "The stage you are building the front end for")
+    }
+    stages {
+        stage("Build") {
+            agent {
+                ecs {
+                    inheritFrom "transfer-frontend"
+                }
+            }
+            steps {
+                dir("api-update") {
+                    sh "sbt test antivirus/assembly"
+                    sh "mv antivirus/target/scala-2.13/antivirus-assembly-*.jar antivirus/target/scala-2.13/antivirus.jar"
```

Comment:
Sorry, only just noticed that this is deploying the antivirus jar. How does the API update Lambda get deployed? Do they all get deployed by the same Jenkinsfile, or can we deploy them separately? I suspect that would be useful, so that we can keep deployments as small as possible.

Reply:
This isn't the antivirus jar for the antivirus checks, we don't have a deployment pipeline for that yet because it's done in a separate ticket. This is deploying the antivirus api update lambda (because we're doing one lambda for each api update) It's just badly named, I'll change the name to antivirus-api-update or something better.

In terms of when we do the checksum api update lambda, I'm not sure whether it will all get put into this Jenkinsfile or if we'll do separate ones but we can cross that bridge when we get to it.

Reply:
Sorry, I'd forgotten that we were going to have multiple update lambdas. Renaming it sounds like a good idea, thanks.